dask_sql/mappings.py

```diff
@@ -177,6 +178,11 @@ def sql_to_python_value(sql_type: "SqlTypeName", literal_value: Any) -> Any:
         # Calcite will always convert INTERVAL types except YEAR, QUATER, MONTH to milliseconds
         # Issue: if sql_type is INTERVAL MICROSECOND, and value <= 1000, literal_value will be rounded to 0
         return np.timedelta64(literal_value, "ms")
+    elif sql_type == SqlTypeName.INTERVAL_MONTH_DAY_NANOSECOND:
+        # DataFusion assumes 30 days per month. Therefore we multiply number of months by 30 and add to days
+        return np.timedelta64(
+            (literal_value[0] * 30) + literal_value[1], "D"
+        ) + np.timedelta64(literal_value[2], "ns")
```

Comment:
This fix makes sense to me - two things I'm interested in:

- is there any way to hit the cases for `INTERVAL` and `INTERVAL_DAY` directly above this? Wasn't able to reach them with our standard interval tests, interested in what CI would look like if we remove them (and maybe also associated plumbing) altogether
- with this change, we might want to consider modifying @sarahyurick's handling for months/quarters in `DatetimeSubOperation` to make the same assumption around a uniform 30 day month for sake of consistency

Reply:
It is possible for `INTERVAL` or `INTERVAL_DAY` cases to be hit. They are still variants of DataFusion. I think we should leave those as are.

Agree on @sarahyurick logic though. I will make the change to make it assume 30 days for consistency sake.